x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/search_ui_logic.ts

```diff
@@ -25,7 +25,7 @@ interface SearchUIActions {
   onFacetFieldsChange(facetFields: string[]): { facetFields: string[] };
   onSortFieldsChange(sortFields: string[]): { sortFields: string[] };
   onTitleFieldChange(titleField: string): { titleField: string };
-  onURLFieldChange(urlField: string): { urlField: string };
+  onUrlFieldChange(urlField: string): { urlField: string };
```

Comment:
Nice! I'll totally confess I never really know when to capitalize acronomys/initials in camelCasing vs not. #YOLO

Reply:
Yeah me neither. I changed this for consistency mainly because the actual value it's updating uses camel casing.